src/EventListener/LogRevisionsListener.php

```diff
@@ -608,13 +612,18 @@ private function saveRevisionEntityData(EntityManagerInterface $em, ClassMetadat
     }
 
     /**
-     * @param array<string, mixed>  $assoc
-     * @param array<string, mixed>  $entityData
+     * @param array<string, mixed> $assoc
+     * @param array<string, mixed> $entityData
      * @param ClassMetadata<object> $class
      * @param ClassMetadata<object> $targetClass
+     * @throws Exception
      */
     private function recordRevisionForManyToManyEntity(object $relatedEntity, EntityManagerInterface $em, string $revType, array $entityData, array $assoc, ClassMetadata $class, ClassMetadata $targetClass): void
     {
+        if ($this->config->areAssociationsDisabled()) {
+            throw new \RuntimeException('This method is unreachable because option "disable_associations" is enabled.');
+        }
```

Comment:
This exception does not seem needed to me. You can remove it.

Reply:
I have made the indicated changes in the latest comitment